doc/user_guide/user_guide.md

```diff
@@ -5,21 +5,16 @@ This library lets you define errors with a uniform set of attributes.
 Furthermore, the error message  is implemented to be parseable, 
 so that you can extract an error catalog from the code.
 
-## Attributes of Error Builder
+## Creating an Error Object
 
-Error messages are built in `ExaError` using the following predefined attributes. 
-All attributes of the error object except the error code are optional. Thus, the 
-`ExaError` will still work, even if the other attributes are not provided. Please 
-keep in mind that error-code should satisfy error-code format 
-(see [error-code](#error-code)).
+Error objects are built in using the fuction `ExaError`. 
+Please keep in mind that error-code should satisfy error-code format (see [code](#code)).
 
 Flexibility is provided by introducing placeholder parameters to the error 
-message in two ways: (1) via the `parameter` attribute and (2) with positional 
-arguments. Furthermore, placeholders without parameters are filled with <null>.
+message.
 
```

Comment:
Add an example call for the `ExaError()` with placeholders for the parameters, e.g.:
```
err = ExaError("<code>", "message", "mitigation", <param dict>)
```
Then it's easier to understand the following sections about each parameter

Reply:
also see https://github.com/exasol/error-reporting-python/issues/25